components/molecules/ListCard/list-card.tsx

```diff
@@ -26,7 +27,15 @@ const ListCard = ({ list, handleOnDeleteClick }: ListCardProps) => {
           <div className="flex items-center gap-4 lg:items-center ">
             <div className="w-4 h-4 rounded-full bg-light-orange-10"></div>
             <div className="flex justify-between text-xl text-light-slate-12">
-              <Link href={`/lists/${list.id}/overview`}>{list.name}</Link>
+              <Link
+                href={
+                  workspaceId
+                    ? `/workspaces/${workspaceId}/contributor-insights/${list.id}/overview`
+                    : `/lists/${list.id}/overview`
```

Comment:
The old URLs are there temporarily until we're all in on workspaces?

Reply:
Yes, that's correct. This is used on the old and new pages